root/static/scripts/common/linkedEntities.js

```diff
@@ -15,22 +15,23 @@
 /*::
 type LinkedEntities = {
   artist_type: {
-    +[number]: ArtistTypeT,
+    [number]: ArtistTypeT,
```

Comment:
Not newly added, ofc, but these probably should be id: number now, even if the eslint rule doesn't detect them.

Reply:
Pushed a commit to add those.